src/Http/Http.Results/src/TypedResults.cs

```diff
@@ -0,0 +1,860 @@
+    /// <returns>The created <see cref="ForbidHttpResult"/> for the response.</returns>
+    public static ForbidHttpResult Forbid(AuthenticationProperties? properties = null, IList<string>? authenticationSchemes = null)
+        => new(authenticationSchemes: authenticationSchemes ?? Array.Empty<string>(), properties);
+
+    /// <summary>
+    /// Creates an <see cref="SignInHttpResult"/> that on execution invokes <see cref="AuthenticationHttpContextExtensions.SignInAsync(HttpContext, string?, ClaimsPrincipal, AuthenticationProperties?)" />.
+    /// </summary>
+    /// <param name="principal">The <see cref="ClaimsPrincipal"/> containing the user claims.</param>
+    /// <param name="properties"><see cref="AuthenticationProperties"/> used to perform the sign-in operation.</param>
+    /// <param name="authenticationScheme">The authentication scheme to use for the sign-in operation.</param>
+    /// <returns>The created <see cref="SignInHttpResult"/> for the response.</returns>
+    public static SignInHttpResult SignIn(
+        ClaimsPrincipal principal,
+        AuthenticationProperties? properties = null,
+        string? authenticationScheme = null)
+        => new(principal, authenticationScheme, properties);
+
+    /// <summary>
+    /// Creates an <see cref="SignOutHttpResult"/> that on execution invokes <see cref="AuthenticationHttpContextExtensions.SignOutAsync(HttpContext, string?, AuthenticationProperties?)" />.
+    /// </summary>
+    /// <param name="properties"><see cref="AuthenticationProperties"/> used to perform the sign-out operation.</param>
+    /// <param name="authenticationSchemes">The authentication scheme to use for the sign-out operation.</param>
+    /// <returns>The created <see cref="SignOutHttpResult"/> for the response.</returns>
+    public static SignOutHttpResult SignOut(AuthenticationProperties? properties = null, IList<string>? authenticationSchemes = null)
+        => new(authenticationSchemes ?? Array.Empty<string>(), properties);
+
+    /// <summary>
+    /// Writes the <paramref name="content"/> string to the HTTP response.
+    /// <para>
+    /// This is an alias for <see cref="Text(string, string?, Encoding?)"/>.
+    /// </para>
+    /// </summary>
+    /// <remarks>
+    /// If encoding is provided by both the 'charset' and the <paramref name="contentEncoding"/> parameters, then
+    /// the <paramref name="contentEncoding"/> parameter is chosen as the final encoding.
+    /// </remarks>
+    /// <param name="content">The content to write to the response.</param>
+    /// <param name="contentType">The content type (MIME type).</param>
+    /// <param name="contentEncoding">The content encoding.</param>
+    /// <returns>The created <see cref="ContentHttpResult"/> object for the response.</returns>
+    public static ContentHttpResult Content(string content, string? contentType = null, Encoding? contentEncoding = null)
```

Comment:
Content and Text are the exact same thing, why do we have both?

Reply:
@brunolins16 do you recall why this was the case?

Reply:
Seems it's like this in `Results` too: https://github.com/dotnet/aspnetcore/blob/main/src/Http/Http.Results/src/Results.cs#L91

Reply:
We do have they in the Results class before, so, I just move the same here. But, I do not have history, but i feel that it was there just for easier discoverability for users and it also includes this comment:

```
/// <para>
/// This is an alias for <see cref="Text(string, string?, Encoding?)"/>.
/// </para>
```

The same happens for `File` and `Bytes` methods